topologies/binding/binding.go

```diff
@@ -165,6 +168,30 @@ func (d *staticDUT) DialCLI(ctx context.Context, opts ...grpc.DialOption) (bindi
 	return newCLI(sc)
 }
 
+func (a *staticATE) DialGNMI(ctx context.Context, opts ...grpc.DialOption) (gpb.GNMIClient, error) {
+	if a.dev.Gnmi == nil {
+		return nil, fmt.Errorf("gnmi must be configured in ate binding")
+	}
+	addr := a.dev.Gnmi.Target
+	opts = append(opts, grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{InsecureSkipVerify: true}))) // NOLINT
+	conn, err := grpc.DialContext(ctx, addr, opts...)
+	if err != nil {
+		return nil, fmt.Errorf("DialContext(ctx, %s, %v): %w", addr, opts, err)
+	}
+	return gpb.NewGNMIClient(conn), nil
+}
+
+func (a *staticATE) DialOTG(context.Context) (gosnappi.GosnappiApi, error) {
+	if a.dev.Otg == nil {
+		return nil, fmt.Errorf("otg must be configured in ate binding")
+	}
+	api := gosnappi.NewApi()
+	api.NewGrpcTransport().
+		SetLocation(a.dev.Otg.Target).
+		SetRequestTimeout(30 * time.Second)
```

Comment:
If request timeout is necessary, let's add that as a binding dial option so all gRPC-based protocols can use it.